### PR TITLE
Remove unused resource package

### PR DESCRIPTION
**Pull request by MarcelMue**
This got moved to `pkg/release` - we remove it here to get an better overview of what is actually used.

### CODE REVIEW

File: pkg/release/chart_info.go [hunk 1]
```diff
@@ -22,6 +22,14 @@ func NewStableChartInfo(name string) ChartInfo {
 	}
 }
 
+func NewChannelChartInfo(name string, channel string) ChartInfo {
```
Comment (MarcelMue) [inline pkg/release/chart_info.go]: Needed for e2etests as it uses channels.

**Review by xh3b4sd — APPROVED** [review]

**Review by kopiczko — APPROVED** [review]

**Review by rossf7 — APPROVED** [review]

**Comment by sslavic** [comment]
Rebased on top of latest master.

**Comment by tomahawk28** [comment]
`chart-operator` is using `resource` pkg still, and it has to move to `pkg/release`. 
@MarcelMue How did you check this? Any migration did you do before you do this PR, so I can also look into it?

**Comment by MarcelMue** [comment]
@tomahawk28 I did check this by vendoring it but it was in April. This was merged by Stevo - My assumption would be that you can use the new functions in place of the old ones. Uncertain as I would need to take a deeper look to get back into it.

**Pull request merged**